svf/lib/MemoryModel/PointerAnalysisImpl.cpp

```diff
@@ -355,6 +371,25 @@ bool BVDataPTAImpl::readFromFile(const string& filename)
     for (auto t: nodePtsMap)
         ptD->unionPts(t.first, strPtsMap[t.second]);
 
+    SVFIR::NodeOffsetMap &gepObjVarMap = pag->getGepObjNodeMap();
+    while (F.good())
+    {
+        getline(F, line);
+        if (line == "------")     break;
+        // Parse a single line in the form of "ID baseNodeID offset"
+        istringstream ss(line);
+        NodeID base;
+        size_t offset;
+        NodeID id;
+        ss >> base >> offset >>id;
+        SVFIR::NodeOffsetMap::const_iterator iter = gepObjVarMap.find(std::make_pair(base, offset));
+        if (iter == gepObjVarMap.end())
+        {
+            pag->readGepObjNodeFromFile(base, offset, id);
+        }
+        
+    }
+
```

Comment:
ln256 - ln266 and ln 393 - ln 408 may be able to be removed since we are now writing and reading all GepObjVarNode to the file via the GepObjVarMap.